programs/ssl/ssl_client2.c

```diff
@@ -1663,13 +1706,24 @@ int main( int argc, char *argv[] )
     fflush( stdout );
 
     mbedtls_entropy_init( &entropy );
-    if( ( ret = mbedtls_ctr_drbg_seed( &ctr_drbg, mbedtls_entropy_func,
-                                       &entropy, (const unsigned char *) pers,
-                                       strlen( pers ) ) ) != 0 )
-    {
-        mbedtls_printf( " failed\n  ! mbedtls_ctr_drbg_seed returned -0x%x\n",
-                        -ret );
-        goto exit;
+    if (opt.reproducible) {
+        if( ( ret = mbedtls_ctr_drbg_seed( &ctr_drbg, dummy_entropy,
+                                          &entropy, (const unsigned char *) pers,
+                                          strlen( pers ) ) ) != 0 )
```

Comment:
Style: Please fix alignment of function parameters here.

Reply:
ok